src/index.js

```diff
@@ -1,5 +1,23 @@
 import './styles/main.css';
 import '@fortawesome/fontawesome-free/js/all.js';
 import retriveAllData from './module/dom-utils.js';
+import { createPopUp, clearPopup, setCord } from './modules/popUp_element.js';
 
-window.onload = () => retriveAllData();
\ No newline at end of file
+const main = document.querySelector('.main');
+
+window.onload = () => retriveAllData();
+
+// Event to create and delete pop-up
+main.addEventListener('click', (e) => {
+  // Create pop-up
+  if (e.target.nodeName === 'BUTTON') {
```

Comment:
it is better to move this code base to display the data function which is in the dom-util.js file.